media/js/mozorg/contribute/contribute-2015-ga.js

```diff
@@ -47,50 +31,6 @@ $(function () {
         });
     });
 
-    // Track Mozillian story clicks on the landing page
-    $('.landing-stories .person .url').each(function() {
-        var person = $(this).parents('.person').find('.fn').text();
-
-        $(this).attr({
-            'data-element-location': person,
-            'data-element-action': 'How Mozillians Help Every Day',
-            'data-tracking-flag': 'contribute-landing'
-        });
-    });
-
-    // Track Mozillian story clicks on story pages
-    $('.stories-other .person .url').each(function() {
-        var person = $(this).parents('.person').find('.fn').text();
-
-        $(this).attr({
-            'data-element-location': person,
-            'data-element-action': 'Meet a few more Mozillians',
-            'data-tracking-flag': 'mozillians'
-        });
-    });
-
-    // Track Twitter hashtag clicks on story pages
-    $('.stories-other .section-tagline a').attr({
-        'data-element-action': 'twitter search link',
-        'data-element-location': '#IAmAMozillian',
-        'data-tracking-flag': 'mozillians'
-    });
-
-    // Track Mozillian personal links
-    $('.story-links a').each(function() {
-        var person = $('.story-title .name').text();
-        var link = $(this).prop('class');
-
-        $(this).attr({
-            'data-element-location': person + ' - ' + link,
-            'data-element-action': 'social button click',
-            'data-tracking-flag': 'mozillians'
-        });
-    });
-
-    // Track other actions on landing page
-    $('.landing-notready .other-actions a').attr('data-element-location', 'not ready');
-
     // Track other actions on confirmation page
     $('#thankyou .other-actions a').each(function() {
```

Comment:
Yes, that's what I'm suggesting. Ideally this kind of thing could bring the template directly.